scripts/net/run-sample-tests.sh

```diff
@@ -164,7 +164,7 @@ start_zephyr ()
     fi
```

Comment:
Why do we need / want to use 64-bit native_posix? That info would be nice to have in commit msg.

Reply:
Actually, we don't. Let's skip this patch.